### ISSUE TITLE
Multi user, let not delete anyone other files

### ISSUE THREAD

**Issue by scriptzteam** [asapach/peerflix-server#99]
For example if i share the url anyone can delete stop my torrents, is there any way to create per user or ip or session id like http://someip.org:9000/#session_id/ so only those with session id can manage and see thier torrents?

THX!

**Comment by asapach** [comment]
I feel like user management, authentication, authorization, etc. is beyond the scope of this project. You can try setting up a reverse proxy (like nginx) in front of the server to handle auth and sessions.

**Comment by scriptzteam** [comment]
nono i mean like ip based check or session check maybe ?

**Comment by asapach** [comment]
Session requires authentication, which in turn requires user management, etc. See the discussion in #77

**Comment by asapach** [comment]
Oh and if you want IP-based security, you can either setup [your firewall](http://unix.stackexchange.com/questions/11851/iptables-allow-certain-ips-and-block-all-other-connection) or configure a reverse proxy: [Apache](https://httpd.apache.org/docs/current/howto/access.html) or [nginx](http://nginx.org/en/docs/http/ngx_http_access_module.html)

**Comment by ViperCode7** [comment]
I have to agree.... the most that bothers me is when some nut job deletes all torrents so he can add his.

What scriptzteam wants is a session based authorization scheme. For instance when a user accesses the site for the first time he gets a session ID based on a hash of his IP and some salt, unix time etc. Adding a torrent will be marked as removable only by that session ID, a timeout of hours or days (if there's already one implemented) or by the owner of the site. 

What you've suggested doesn't hinder anyone from removing the torrents. What we wants if to separate superuser rights on what we add vs other users. I should not be able to remove someone else's torrents, nor should he mine.

**Comment by scriptzteam** [comment]
torx.cat works this way ;)

**Comment by ViperCode7** [comment]
When it works... for me it doesn't and peerflix-server is more popular, way way more sites with it.
